package.json

```
@@ -24,7 +24,7 @@
     "node": ">= 10.0.0"
   },
   "dependencies": {
-    "axe-core": "4.4.1",
+    "axe-core": "4.5.0",
```

This is now released and includes fix for https://github.com/dequelabs/axe-core/issues/3753.

```suggestion
    "axe-core": "4.5.1",
```

---

@AriPerkkio thank you for the update :) I updated the PR to also include the corresponding change in the `package-lock.json`